code/game/machinery/doors/blast_door.dm

```diff
@@ -123,15 +128,33 @@
 					usr << "<span class='warning'>You don't have enough sheets to repair this! You need at least [amt] sheets.</span>"
 
 
+		else if(src.density)
+			var/obj/item/weapon/W = C
+			user.setClickCooldown(DEFAULT_ATTACK_COOLDOWN*2)
```

Comment:
Doing so would halve the time, and we're back to them breaking out in 30 seconds, just with fewer strikes.